tools/travis/integration.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+# Build script for Travis-CI.
+
+SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+ROOTDIR="$SCRIPTDIR/../.."
+WHISKDIR="$ROOTDIR/../openwhisk"
+UTILDIR="$ROOTDIR/../incubator-openwhisk-utilities"
+
+# run scancode
+cd $UTILDIR
+scancode/scanCode.py $ROOTDIR
+
+# TODO run jshint
+
+# Install OpenWhisk
+cd $WHISKDIR/ansible
+
+ANSIBLE_CMD="ansible-playbook -i environments/local"
```

Comment:
I think we can use the images from docker hub making travis builds super fast by not needing to build the images need to pass the option `-e docker_image_prefix=openwhisk`